src/internal/serviceenv/config.go

```diff
@@ -21,8 +21,8 @@ type GlobalConfiguration struct {
 	Namespace                      string `env:"PACH_NAMESPACE,default=default"`
 	StorageRoot                    string `env:"PACH_ROOT,default=/pach"`
 	GCPercent                      int    `env:"GC_PERCENT,default=50"`
-	LokiHost                       string `env:"LOKI_SERVICE_HOST"`
-	LokiPort                       string `env:"LOKI_SERVICE_PORT"`
+	LokiHost                       string `env:"PACHYDERM_LOKI_SERVICE_HOST"`
```

Comment:
One thing I don't like about this right now, is that it forces the client to use our subchart, and restricts them from separately deploying loki.

Reply:
we could pass a name override to the loki chart perhaps

Reply:
how does that work?

Reply:
so we can repurpose these config values to LokiHost, LokiPort to store the env-var name instead - maybe change their names to LokiHostVar and LokiPortVar